secboot/encrypt.go

```diff
@@ -38,6 +38,9 @@ const (
 	//      github.com/snapcore/secboot/crypto.go:"type RecoveryKey"
 	// Size of the recovery key.
 	recoveryKeySize = 16
+
+	// The auxiliary key is used to bind keys to models
+	auxKeySize = 32
```

Comment:
I noticed whilst reviewing this that encryptionKeySize is still set to 64, which isn't necessary - the overall security strength of the system is limited to 32 bytes anyway (AES-256 which is used to protect the key uses a 32 byte key), so there's no additional benefit of creating a 64 byte key.